packages/web-components/src/spinner/spinner.options.ts

```diff
@@ -0,0 +1,43 @@
+export type Values<T> = T[keyof T];
+/**
+ * Spinner size options
+ * @public
+ */
+export const SpinnerSize = {
+  /**
+   * A Spinner's size can be small, tiny, extra-small, medium, large, extra-large, or huge
+   *
+   */
+  small: 'small',
+  tiny: 'tiny',
+  'extra-small': 'extra-small',
```

Comment:
I'd suggest that we avoid keys as strings because the implementation of this is going to create an inconsistent experience. The important thing is that the _value_ is consistent for our types. In terms of keys, I'd suggest we follow our property convention of `camelCase`. With a string like this, in order to use it a developer is going to need to do:

```ts
const foo: SpinnerSize = SpinnerSize['extra-small'];
```

What I think we'll want (and what will be consistent between exported instances in FAST is:
```ts
const foo SpinnerSize = SpinnerSize.extraSmall;
```
This will give us a consistent application and use across. Similarly, when we hit numbers, we tend to prefix the actual key with an underscore. So if we look at Text size, you'll notice that the value is consistent but due to constraints w/ TS we can't use a number as a key. So to consistently use dot notation, we would have `TextSize._100` rather than TextSize['100']. Example below of this consideration as well:

<img width="323" alt="image" src="https://user-images.githubusercontent.com/13071055/214166582-9b13a22f-ed09-4f4f-a4ed-9c18af680a3f.png">